modules/extensions-xml-support/src/main/java/org/mule/runtime/extension/internal/ast/MacroExpansionModuleModel.java

```diff
@@ -160,6 +253,26 @@ private ArtifactAst expand(List<ComponentAst> moduleComponentModels, Set<String>
                            comp -> false);
   }
 
+  private boolean existOperationThatUsesImplicitConfiguration() {
+    return applicationModel.topLevelComponentsStream()
+        .filter(componentAst -> componentAst.getIdentifier().getName().equals("flow")
+            || componentAst.getIdentifier().getName().equals("sub-flow"))
```

Comment:
put the flow/subflow constants in some static array/list so that if any other needs to be added it just needs to be added there.